src/linter/phpdoc_util.go

```diff
@@ -100,3 +101,18 @@ func parseClassPHPDocProperty(ctx *rootContext, result *classPhpDocParseResult,
 		AccessLevel: meta.Public,
 	}
 }
+
+func parseClassPHPDocMixin(ctx *rootContext, result *classPhpDocParseResult, part *phpdoc.RawCommentPart) {
+	params := part.Params
+
+	if len(params) == 0 {
+		return
+	}
+
+	param := params[0]
+	if !strings.HasPrefix(param, `\`) {
```

Comment:
Usually the classes mentioned in phpdoc are defined inside a namespace. You need to resolve the class name the same way it is implemented for @return types, for example.

Reply:
Sorry if I misunderstood you. I tried testing the following:

```php
// file1.php
namespace QQ;

class SomeQQClass {
  /** */
  public function methodQQ()
  {
    echo "";
  }
}
```
```php
// file2.php
/**
 * @mixin QQ\SomeQQClass
 */
class SomeClass {
  /** */
  public function method()
  {
    echo $this->methodQQ();
  }
}
```

This works correctly, there are no errors.

Did you mean that?

Reply:
You can also write the following thing, for example:

```
<?php
namespace QQ;

/** @mixin SomeQQClass */
class A {
}
```

SomeQQClass here actually means \QQ\SomeQQClass

Reply:
Thank you I got it.

Reply:
That was only one example :). I suggest that you use the existing functions that resolve types in phpdoc instead. There are more cases that should be properly handled, e.g.

```
<?php

namespace QQ;

use A as B;

class C {
/** @mixin A */
// This actually means mixin \QQ\B
}
```